runtime/commonTest/src/kotlinx/serialization/protobuf/ProtobufReaderTest.kt

```diff
@@ -14,49 +14,49 @@ class ProtobufReaderTest {
 
     @Test
     fun readSimpleObject() {
-        ProtoBuf.loads<TestInt>(TestInt.serializer(), "08ab02") shouldBe t1
+        ProtoBuf.loads(TestInt.serializer(), "08ab02") shouldBe t1
```

Comment:
Removing the type parameters here is valid in principle, but shouldn't really be part of this pull request/patch set.